Automattic/wp-calypso
# A4A: Fixing pagination on the sites dashboard

Related to https://github.com/Automattic/automattic-for-agencies-dev/issues/201

## Proposed Changes

* This PR adds styling to ensure pagination displays on the sites dashboard on different screen sizes and with the preview pane, fixed at the bottom of the dashboard.
* This PR also fixes a scroll issue on screen sizes between 961 to 1400px where it wasn't possible to scroll down a list of sites in the main sites dashboard view.

Figma designs: 1UpoqS1KkCtLXVX68TbMSM-fi-6144_70981

## Testing Instructions

For the below testing steps, make sure to run `yarn start-a8c-for-agencies` locally and then visit `http://agencies.localhost:3000/sites`.

* To test pagination, use the 'View options' icon and select a rows per page option that will allow you see your sites paginated (you may need to add extra Jetpack connected sites to have more than 10 sites for example).
* Then, test pagination with different screen sizes plus on mobile, as well as with the preview pane open. The Figma link above shows some examples of how the pagination should look.

Screenshots:
<img width="1256" alt="Screenshot 2024-04-04 at 21 19 24" src="https://github.com/Automattic/wp-calypso/assets/16754605/5f9b5a77-edaf-4083-ad0f-a449f841aa1f">

On mobile:
<img width="308" alt="Screenshot 2024-04-04 at 21 20 38" src="https://github.com/Automattic/wp-calypso/assets/16754605/702d9906-4537-4e92-bdb4-74560b67e780">



## Pre-merge Checklist
